src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,21 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @ManyToMany
     private List<Actor> actors;
```

Comment:
Initializing the 'actors' list in the entity will prevent NullPointerExceptions when adding to the collection. Consider initializing it inline (e.g., private List<Actor> actors = new ArrayList<>();) or in the default constructor.